tests/Unit/Utilities/DeclarationBlockParserTest.php

```diff
@@ -30,6 +30,10 @@ public function providePropertyNameAndExpectedNormalization(): array
                 'name' => '--text-color',
                 'expect' => '--text-color',
             ],
+            'custom property with numbers' => [
+                'name' => '--base-size-16',
+                'expect' => '--base-size-16',
+            ],
```

Comment:
The code change also allows underscores in the property name, so I think that should be tested too.